Source/WebSocket.swift

```diff
@@ -860,8 +879,9 @@ open class WebSocket : NSObject, StreamDelegate {
     /**
      Used to write things to the stream
      */
-    private func dequeueWrite(_ data: Data, code: OpCode, writeCompletion: (() -> ())? = nil) {
+    private func dequeueWrite(_ data: Data, code: OpCode, priority: Operation.QueuePriority = .normal, writeCompletion: (() -> ())? = nil) {
```

Comment:
With respect to these queue does this mean that if I send a low priority then a high priority the high priority would be sent first or is the order maintained?